openedx/core/djangoapps/user_api/accounts/tests/utils.py

```diff
@@ -0,0 +1,20 @@
+"""
+Utilities for testing the accounts API.
+"""
+from django.test import TestCase
+from django.test.utils import override_settings
+
+
+@override_settings(
```

Comment:
An alternative to this mixin is to just put the configuration in lms/envs/test.py. I think that would be better because that is where we usually put test-specific configuration, this mixin does nothing else.